plugins/check_curl.c

```diff
@@ -1133,6 +1133,11 @@ redir (curlhelp_write_curlbuf* header_buf)
     &status_line.http_major, &status_line.http_minor, &status_line.http_code, &status_line.msg, &msglen,
     headers, &nof_headers, 0);
 
+	if (res == -1) {
```

Comment:
Why only here? There is also one in 2304, 2362?
phr_parse_response is not returning -1 on error. What should be done on -2 (partial response)?

Reply:
I don't know :-| What would you suggest? Die in all non-OK cases?